config/__init__.py

```diff
@@ -0,0 +1,21 @@
+# flake8: noqa
+from os import environ
+
+FLASK_ENV = environ.get("FLASK_ENV")
```

Comment:
🔧 See the `getenv` [method in the standard library](https://docs.python.org/3/library/os.html#os.getenv) :)